yang/frr-isisd.yang

```diff
@@ -104,6 +104,16 @@ module frr-isisd {
         description
           "This enum indicates capability for both levels.";
       }
+        enum "level2_to_level1" {
```

Comment:
There are no such levels in IS-IS, it shouldn't be modeled like this. More comments below.